test/AutoDiff/validation-test/modify_accessor.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %target-run-simple-swift
```

Comment:
The example here is not directly related, though the autodiff code currently is the only producer of `partial_apply` of coroutines in the codebase (though, nothing prohibits this per SIL spec).